### PR TITLE
Enable blockWriter write pressure in stability test

### PR DESCRIPTION
**Pull request by aylei**
* Enable blockWriter
* Make blockWriter configurable

Result:
image

Signed-off-by: Aylei <rayingecho@gmail.com>

### CODE REVIEW

**Comment by aylei** [comment]
@weekface @xiaojingchen PTAL

File: tests/cmd/stability/main.go [hunk 1]
```diff
@@ -154,10 +154,16 @@ func main() {
 	oa.CheckTidbClusterStatusOrDie(cluster1)
 	oa.CheckTidbClusterStatusOrDie(cluster2)
 
-	//go func() {
-	//	oa.BeginInsertDataTo(cluster1)
-	//	oa.BeginInsertDataTo(cluster2)
-	//}()
+	defer func() {
```
Comment (xiaojingchen) [inline tests/cmd/stability/main.go]: why not split two defer function?

Comment (aylei) [reply tests/cmd/stability/main.go]: What's the point of split two defer functions?

Comment (aylei) [reply tests/cmd/stability/main.go]: I get it

Comment (xiaojingchen) [reply tests/cmd/stability/main.go]: just avoid to executing useless defer，in the other hand the errors in defer can be confused with real errors

File: tests/backup/backupcase.go [hunk 2]
```diff
@@ -36,11 +36,16 @@ func NewBackupCase(operator tests.OperatorActions, srcCluster *tests.TidbCluster
...
+
+	// pause write pressure during backup
+	defer func() {
+		go func() {
+			if err := bc.operator.BeginInsertDataTo(bc.srcCluster); err != nil {
+				glog.Errorf("cluster:[%s] begin insert data failed,error: %v", bc.srcCluster.ClusterName, err)
+			}
+		}()
+	}()
+	bc.operator.StopInsertDataTo(bc.srcCluster)
```
Comment (xiaojingchen) [inline tests/backup/backupcase.go]: ditto

**Comment by aylei** [comment]
@xiaojingchen Is this looks fine now?
&
/run-e2e-tests

**Pull request merged**